Development

JonWuIDM opened the pull request:
Here's the folder.

JonWuIDM committed:
Ideas

JonWuIDM committed:
Links

JonWuIDM committed:
Added a Test Run File

Along with custom pixel art ~testing~

JonWuIDM commented:
Added a test run for images I will be using.

JonWuIDM committed:
Notes

Added Sound and More graphics

JonWuIDM commented:
Pull Request

JonWuIDM committed:
Notes

Updated, added more basic functionality. Moves! Need to find a way to
move diagonally.

LadyK commented:
Control is a big issue here. I'd look into switch statements and variables to hold various states, and checking systems to execute based on certain conditions. Look at the example code I just sent out. Your internal architect needs to be tight. Work slowly, checking things again other things, versus a big code dump. Also, look into controlling your sound. One of those tracks is almost 3 minutes. You might want to play a clip or between various points, versus the entire track. Again, control and manipulation is critical. Get that together before moving this forward.

JonWuIDM committed:
Notes

Updated

JonWuIDM committed:
Notes

Some things work, some don't. Health bar and the horizontal scrolling
need fixing.
